common/aws/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -166,26 +172,29 @@ func (this *CloudWatchLogs) DescribeLogStreams(logGroupName, orderBy string) ([]
 	}
 
 	input := &cloudwatchlogs.DescribeLogStreamsInput{
-		LogGroupName: aws.String(logGroupName),
-		OrderBy:      aws.String(orderBy),
+		LogGroupName:        aws.String(logGroupName),
+		OrderBy:             aws.String(orderBy),
+		Descending:          aws.Bool(true),
```

Comment:
I may need to test again, but w/o this I'm pretty sure you get oldest logs first. For example, I wasn't getting logs for the `api` service because there are like 6k older streams in front of it.